devpi/entrypoint.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+# Adapted from: https://github.com/LordGaav/docker-devpi
+
+function generate_password() {
+    # We disable exit on error because we close the pipe
+    # when we have enough characters, which results in a
+    # non-zero exit status
+    set +e
+    tr -cd '[:alnum:]' < /dev/urandom | fold -w30 | head -n1 | tr -cd '[:alnum:]'
+    set -e
+}
+
+function kill_devpi() {
+    _PID=$(pgrep devpi-server)
+    echo "ENTRYPOINT: Sending SIGTERM to PID $_PID"
+    kill -SIGTERM "$_PID"
+}
+
+if [ "${1:-}" == "bash" ]; then
+    exec "$@"
+fi
+
+DEVPI_ROOT_PASSWORD="${DEVPI_ROOT_PASSWORD:-}"
+if [ -f "$DEVPI_SERVER_ROOT/.root_password" ]; then
+    DEVPI_ROOT_PASSWORD=$(cat "$DEVPI_SERVER_ROOT/.root_password")
```

Comment:
All of this can be removed in our case probably. This script can just expect `DEVPI_ROOT_PASSWORD` env variable to be set?.